repository/model/project.go

```diff
@@ -14,7 +16,10 @@ type Project struct {
 	Name                      string                    `db:"name"`
 	SlackChannelID            string                    `db:"slack_channel_id"`
 	ReleaseNotificationConfig ReleaseNotificationConfig `db:"release_notification_config"`
-	GithubRepositoryURL       string                    `db:"github_repository_url"`
+	GithubRepositoryURL       string                    `db:"github_repository_url"` // TODO remove this field
+	GithubOwnerSlug           sql.NullString            `db:"github_owner_slug"`
+	GithubRepoSlug            sql.NullString            `db:"github_repo_slug"`
+	GithubRepoURL             sql.NullString            `db:"github_repo_url"`
```

Comment:
The RepoURL will be saved to the database because it is directly returned by the GitHub client. This avoids complicating the implementation with dynamic URL generation.

When the GitHub repo URL needs updating, you can simply call `/projects/{id}/github-repository` to update the connection.

-----

Yes it is make sense to generate git tag url dynamically but not the repo url.